index.html

```diff
@@ -15,19 +15,39 @@
   </head>
   <body>
     <header class="header-section">
```

Comment:
- [ ] Kindly make the menu links appear in the desktop version to match the Figma design.

| Your design |
| --- | 
| ![image](https://user-images.githubusercontent.com/24703171/215493275-e74180a1-6256-4652-bc26-b9ec8250e34f.png) | 

| Figma design |
| --- | 
![image](https://user-images.githubusercontent.com/24703171/215492450-cf405b0e-d17c-4f71-b401-8ce0bc1a0abe.png) |